janetnim/maintenance-tracker
# #157781455 Design Sign up form

**What does this PR do?**
Create a user/ admin sign up form 

**Description of Task to be completed?**
A sign up form should be designed with username, email and password fields

**How should this be manually tested?**
After adding the html and css files to the UI directory, open the user_signup.html file in your browser to see if all components of the page appear and work as they should

** What are the relevant pivotal tracker stories?**
#157781455

![image](https://user-images.githubusercontent.com/29487740/40603037-6ba22e1e-6263-11e8-9856-26ff11248508.png)
